client/dogs/FormPage.js

```diff
@@ -652,6 +671,69 @@ export default class IndexPage extends React.Component {
             </div>
 
             <div className="txt">
+              <label htmlFor="lookup-rabiesCert" className="txt-l">
+                Rabies Certificate
+              </label>
+              <Dropzone
+                disableClick={false}
+                multiple={false}
+                accept={'image/*'}
+                onDrop={this.onDropRabies.bind(this)}
```

Comment:
Ok great, thanks for pointing that out! All set: https://github.com/CityOfBoston/dog-licenses/pull/31/commits/16639ae7b87a0b6f2603de9e72db8126dae477fc